expan/core/experiment.py

```diff
@@ -73,26 +81,19 @@ def __init__(self, control_variant_name, data, metadata, report_kpi_names=[], de
             self.data.loc[:, name] = eval(re.sub(kpi_name_pattern, r'self.data.\1.astype(float)', formula))
             self.reference_kpis[name] = re.sub(kpi_name_pattern + '/', '', formula)
 
-
-
     def get_kpi_by_name_and_variant(self, name, variant):
         return self.data.reset_index().set_index('variant').loc[variant, name]
 
-
     def __str__(self):
-        # res = super(Experiment, self).__str__()
+        res = super(Experiment, self).__str__()
```

Comment:
commenting that out and assignment res+= produces an error because the variable does not exist. We don't really need the printing, but I decided to leave it, but we need either remove it entirely or just return the string like 
`return '{:d} variants: {}'.format(len(variants),
                                          ', '.join([('*' + k + '*') if (k == self.control_variant_name) else k
                                                    for k in variants]))`

 and remove super because we don't have parenting class anymore.